sources.json

```diff
@@ -581,5 +581,19 @@
     "type": "command",
     "tags": ["development", "tooling", "testing", "browser"],
     "license": "mit"
+  },
+    {
+    "owner": "SAP",
+    "repo": "ui5-tooling-extensions",
+    "subpath": "packages",
+    "subpackages": [
+      {
+        "name": "@ui5/middleware-code-coverage",
```

Comment:
this need to be the folder name "middleware-code-coverage" no the package name
i get the package name from the package.json